client/src/components/PopoverField/PopoverField.js

```diff
@@ -89,6 +100,7 @@ PopoverField.propTypes = {
   container: React.PropTypes.any,
   className: React.PropTypes.string,
   popoverClassName: React.PropTypes.string,
+  buttonSizeClassName: React.PropTypes.string,
```

Comment:
how about a `buttonSize` prop instead and a validation with `PropTypes.oneOf(['xl', 'md', 'sm', 'xs']);`